modules/runner/src/integrationTest/java/org/apache/ignite/internal/runner/app/ItDataSchemaSyncTest.java

```diff
@@ -62,14 +60,9 @@
  */
 @ExtendWith(WorkDirectoryExtension.class)
 public class ItDataSchemaSyncTest extends IgniteAbstractTest {
-    /**
-     * Table name.
-     */
     public static final String TABLE_NAME = "tbl1";
 
-    /**
-     * Nodes bootstrap configuration.
-     */
+    /** Nodes bootstrap configuration. */
```

Comment:
You removed some comments, reformatted others. Why?

Reply:
Please show the rule where this is prohibited.

Reply:
This is not a good answer. I asked "why", please reply. I don't understand this inconsistent approach to changes. Why this comment was important, and other weren't? What's the difference? Why did you touch them in the first place?

Reply:
We discussed it as a team and decided not to remove the changes.